pytorch_lightning/metrics/classification.py

```diff
@@ -52,26 +52,28 @@ class Accuracy(TensorMetric):
     def __init__(
             self,
             num_classes: Optional[int] = None,
-            reduction: str = 'elementwise_mean',
+            class_reduction: str = 'micro',
             reduce_group: Any = None,
             reduce_op: Any = None,
     ):
         """
         Args:
             num_classes: number of classes
-            reduction: a method to reduce metric score over labels (default: takes the mean)
-                Available reduction methods:
-                - elementwise_mean: takes the mean
-                - none: pass array
-                - sum: add elements
+            class_reduction: reduction method for multiclass problems
+
+                - ``'micro'``: calculate metrics globally (default)
+                - ``'macro'``: calculate metrics for each label, and find their unweighted mean.
+                - ``'weighted'``: calculate metrics for each label, and find their unweighted mean.
```

Comment:
isn't this weighted sum??

Reply:
yes of course it is, should be fixed now :]